### ISSUE TITLE
previous issue mentioned tracking and report

### ISSUE THREAD

**Issue by KernilPanik** [simlar/simlar-android#742]
The previous issue went unresponded to, mentioning tracking and Reporting, I would also like to know why audio recording is necessary. Why are these three things needed in an app supposedly built for privacy? They seem very unnecessary, and I'm not going to lie it makes me a little suspicious to use the app, even though I love the premise of it. Can we get an answer to this?

**Comment by BenSartor** [comment]
Hi @KernilPanik,

I have just responded to the issue you mentioned. Simlar is a free time project that has never earned a cent. So answers may take a while.

Audio recording is obviously needed for making a phone call :) However we documented our permissions in the [wiki](https://github.com/simlar/simlar-android/wiki/Simlar-Android-Permissions).

**Comment by Etownsolutionspecialist** [comment]
I have to agree it seems that the intended functionality and main purpose of the app is contradicted by your own permissions list. Its safe to say that your downloads and interest in the app would skyrocket if there was a way to allow the users of the app full permission of what they want to make available with regards to any data collection. I downloaded the app with hopes of using it but cannot justify doing so with the lines of privacy being blurred as it seems they currently are.

**Comment by KernilPanik** [comment]
I was thinking since the source code is up, ripping out that permission,
maybe some refactoring, and trying to see if it works honestly. I don't
think there's a need for recording audio for a phone call, even though
that's the official explanation.
